ams/server/src/main/java/com/netease/arctic/server/optimizing/plan/MixedHivePartitionPlan.java

```diff
@@ -108,7 +110,9 @@ private String constructCustomHiveSubdirectory() {
 
   protected static class MixedHivePartitionEvaluator extends MixedIcebergPartitionEvaluator {
     private final String hiveLocation;
-    private boolean hasNotInHiveLocationFile = false;
+    private boolean hasNotInHiveLocationDataFile = false;
```

Comment:
this is not a comprehensible variable name. make it simple like filesInHiveLocation or dataFilesHasMoved

Reply:
OK, `filesNotInHiveLocation` is more accurate.